packages/caliper-core/lib/common/config/Config.js

```diff
@@ -91,7 +91,8 @@ const keys = {
         Communication: {
             Method: 'caliper-worker-communication-method',
             Address: 'caliper-worker-communication-address',
-        }
+        },
+        MaxTxPromises: 'maxtxpromises'
```

Comment:
Shouldn't the string be `caliper-worker-maxtxpromises`?

Reply:
yes, yes it should (hangs head in shame ... )